introduction.md

```diff
@@ -7,11 +7,10 @@ My favorite "path" is Ruby or Javascript.
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?